crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -0,0 +1,543 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use clap::*;
+use futures::future::try_join_all;
+use futures::future::{join_all, BoxFuture};
+use futures::FutureExt;
+use futures::{stream::FuturesUnordered, StreamExt};
+use rand::seq::IteratorRandom;
+use std::collections::{BTreeMap, HashMap, VecDeque};
+use std::sync::Arc;
+use std::time::Duration;
+use strum_macros::EnumString;
+use sui_config::NetworkConfig;
+use sui_core::authority_aggregator::AuthorityAggregator;
+use sui_core::authority_client::NetworkAuthorityClient;
+use sui_quorum_driver::QuorumDriverHandler;
+use sui_types::base_types::{ObjectID, ObjectRef, SuiAddress};
+use sui_types::crypto::{get_key_pair, EmptySignInfo, KeyPair};
+use sui_types::messages::{
+    ExecuteTransactionRequest, ExecuteTransactionRequestType, ExecuteTransactionResponse,
+    Transaction, TransactionEnvelope,
+};
+use sui_types::object::{Object, Owner};
+use test_utils::authority::{
+    spawn_test_authorities, test_and_configure_authority_configs, test_authority_aggregator,
+};
+use test_utils::messages::{
+    make_counter_create_transaction, make_counter_increment_transaction,
+    make_transfer_object_transaction,
+};
+use test_utils::objects::{
+    generate_gas_object, generate_gas_objects, generate_gas_objects_with_owner,
+};
+use test_utils::transaction::publish_counter_package;
+use tokio::time;
+use tokio::time::Instant;
+use tracing::subscriber::set_global_default;
+use tracing::{debug, error};
+use tracing_subscriber::EnvFilter;
+
+#[derive(Parser)]
+#[clap(name = "Stress Testing Framework")]
+struct Opts {
+    /// Size of the Sui committee.
+    #[clap(long, default_value = "4", global = true)]
+    pub committee_size: usize,
+    /// Target qps
+    #[clap(long, default_value = "1000", global = true)]
+    pub target_qps: u64,
+    /// Number of workers
+    #[clap(long, default_value = "2", global = true)]
+    pub num_workers: u64,
+    /// Max in-flight ratio
+    #[clap(long, default_value = "5", global = true)]
+    pub in_flight_ratio: usize,
+    /// Num of accounts to use for transfer objects
+    #[clap(long, default_value = "5", global = true)]
+    pub num_transfer_accounts: usize,
+    /// Stat collection interval seconds
+    #[clap(long, default_value = "10", global = true)]
+    pub stat_collection_interval: u64,
```

Comment:
Some of these are usize while others are u64, any reason why you picked one vs the other? might cut down on some casts throughout the code if you standardize on one or the other.

Reply:
alright, everything is u64 now. I agree, having a consistent type is much better